src/umpire/tpl/judy/judyL2Array.h

```diff
@@ -217,7 +217,7 @@ class judyL2Array {
         bool removeEntry( JudyKey key ) {
             if( 0 != ( _lastSlot = ( vector ** ) judy_slot( _judyarray, ( const unsigned char * ) &key, _depth * JUDY_key_size ) ) ) {
                 // _lastSlot->~vector(); //for use with placement new
-                // delete _lastSlot;
+                delete *_lastSlot;
```

Comment:
Are you 100% sure this is right?

Reply:
No. All I can say is that it looks right, it prevents a serious memory leak, and the tests pass.